### DISCUSSION TITLE
Discussions tab disappears in private repositories

### DISCUSSION THREAD

**Discussion by doitian** [community/community#4897] [Discussions] [answered]
<img width="1044" alt="2021-08-03_13-11-01" src="https://user-images.githubusercontent.com/35768/127961362-559ac8e2-7594-441b-ab0c-3dbd70691ac4.png">

**Comment by OndrejMus** [comment, marked as answer]
Same issue here. Discussions enabled on repository, available and working through link (https://github.com/organizationname/reponame/discussions) but button in menu is missing

**Reply by gerhardwagner** [reply]
same issue here with organization internal repositories

**Reply by OndrejMus** [reply]
Button appeared again. Seems fine for now

**Comment by PhilDay-CT** [comment]
Us to.
